src/main/java/seedu/address/model/company/exceptions/CompanyNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package seedu.address.model.company.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified company.
+ */
+public class CompanyNotFoundException extends RuntimeException {
```

Comment:
Actually the PersonNotFoundException I got this from also had no constructor, do we want to add one for all such exceptions?

Reply:
It does have a default constructor so I think it is fine. Maybe small nit but perhaps the `}` should be on line 6 as well if you are basing it off `PersonNotFoundException`

Reply:
> Actually the PersonNotFoundException I got this from also had no constructor, do we want to add one for all such exceptions?

You are right, now that I think about it, this is probably because this is an unchecked exception and the program is not expected to recover from it, so no friendly error message is needed. 

I think this conversation can be resolved now.